src/Microsoft.ApplicationInsights/TelemetryClient.cs

```diff
@@ -29,10 +29,18 @@ public sealed class TelemetryClient
 #endif
         private readonly TelemetryConfiguration configuration;
         private string sdkVersion;
-        
+
+#pragma warning disable 612, 618 // TelemetryConfiguration.Active
         /// <summary>
         /// Initializes a new instance of the <see cref="TelemetryClient" /> class. Send telemetry with the active configuration, usually loaded from ApplicationInsights.config.
         /// </summary>
+#if NETSTANDARD1_3 || NETSTANDARD2_0
+        [Obsolete("TelemetryClient parameterless constructor uses TelemetryConfiguration.Active singleton. " +
+                  "We do not recommend using TelemetryConfiguration.Active on .NET Core. " + 
+                  "Instead we suggest using dependency injection pattern. " + 
+                  "See https://docs.microsoft.com/en-us/aspnet/core/fundamentals/dependency-injection?view=aspnetcore-2.2 " + 
+                  "and https://docs.microsoft.com/en-us/azure/azure-monitor/app/asp-net-core for more info")]
```

Comment:
its best to create a new issue in Github repo, and link to the above links from there. Main reason is - the above links are all 'asp.net core' specific, not so much about '.net core console'. So our Github issue can act as a wrapper, and have links to asp.net core with di, .net core with di, or just plain .net core console etc. (if needed).

Also it'd be good place for us to explain the rationale behind this decision in a github issue in this repo.